green_cli/authenticators/wally.py

```diff
@@ -16,8 +18,11 @@ def name(self):
 
     def create(self, session_obj):
         """Create and register a new wallet"""
-        logging.warning("Generating mnemonic using gdk")
-        self._mnemonic = gdk.generate_mnemonic()
+        entropy = secrets.token_bytes(wally.BIP39_ENTROPY_LEN_256)
+        wordlist = wally.bip39_get_wordlist('en')
+        mnemonic = wally.bip39_mnemonic_from_bytes(wordlist, entropy)
```

Comment:
'None' would defaults to wordlist('en') I think - not that there's an issue with being explicit.